user/user.py

```diff
@@ -16,7 +16,8 @@
 
 user_bp = Blueprint('user_bp', __name__,
                     template_folder='templates/user',
```

Comment:
This was a mistake in the original dummy: the paths to the folder should not contain the name of the Blueprint, since this will cause name  conflicts. In short, to prevent naming conflicts do:
template_folder='templates',
static_folder='static',

Note: this changes the locations of the views you load.

Reply:
So all render_template(x) function calls should be updated to render_template('user/x')